src/styles/components/card.scss

```diff
@@ -0,0 +1,116 @@
+.card {
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  transition: transform 0.3s ease-in-out;
+
+
+  &__title {
+    color: $main-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    transition: color 0.3s ease-in-out;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+    cursor: pointer;
+
+    .card__title {
+      color: $card-hover-color;
+    }
+  }
+
+  &__top {
+    padding: 32px 20px 40px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
```

Comment:
Move to the * selector and remove it from all other places.

Reply:
Why should i? Header has different styles
Here is the result of your advice

![image](https://github.com/mate-academy/layout_catalog/assets/106512918/892b5ed2-d391-4188-8ca6-a117ad096980)